Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -352,20 +352,25 @@ private void StandAloneThreadExceptionHandler(object sender, UnhandledExceptionE
         public bool OpenSolution(string solutionFolder)
         {                      
             try
-            {               
+            {
+                Console.WriteLine("Folder in WorkSpace" + solutionFolder);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)